receiver/splunksearchapireceiver/go.mod

```diff
@@ -3,34 +3,65 @@ module github.com/open-telemetry/opentelemtry-collector-contrib/receiver/splunks
 go 1.22.5
 
 require (
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza v0.113.0
 	github.com/stretchr/testify v1.9.0
 	go.opentelemetry.io/collector/component v0.113.0
+	go.opentelemetry.io/collector/confmap v1.20.0
 	go.opentelemetry.io/collector/consumer v0.113.0
+	go.opentelemetry.io/collector/consumer/consumertest v0.113.0
+	go.opentelemetry.io/collector/extension/experimental/storage v0.113.0
+	go.opentelemetry.io/collector/filter v0.114.0
```

Comment:
is this supposed to v0.114.0? I'd imagine a merge from main might smooth out some of these dependency conflicts?

Reply:
I'm not seeing any conflicts personally. Tidy-ing doesn't update these deps. Should I rebase the feature branch to the latest release?

Reply:
Perhaps @dpaasman00 would know better; but I'd imagine that we probably want `go.opentelemetry.io/collector/filter v0.113.0` rather than v0.114.0 but since this is not being merged into main for a little while; I'd imagine its not worth worrying about too much until we want to start entertaining a main merge.